### ISSUE TITLE
Update readme.md

### ISSUE THREAD

**Issue by WebShivam** [rohan-kulkarni-25/Learn-GITHUB#57]
In "Setting Project on Local System" Section We should add one more point after cloning the repo to local system is to `create new branch` and do changes there which will avoid conflicts from `main` branch during pushing

**Comment by rohan-kulkarni-25** [comment]
yes but that will be added seperate section when will add the section of creating branch this will avoid confusion to begineers
